scripts/ansible/oe-windows-acc-setup.yml

```diff
@@ -8,10 +8,10 @@
   tasks:
     - name: OE setup | Set installer URLs from the OE storage account
       set_fact:
-        intel_psw_url:       "https://oejenkins.blob.core.windows.net/oejenkins/Intel%20SGX%20PSW%20for%20Windows%20v2.8.100.2.exe"
-        intel_psw_hash:      "02E5A3E376B450A502F7D79B79A0CCFBCAFA1AFBF15FB7EA6C8830020928D68A"
-        intel_dcap_url:      "https://oejenkins.blob.core.windows.net/oejenkins/Intel%20SGX%20DCAP%20for%20Windows%20v1.7.100.2.exe"
-        intel_dcap_hash:     "5BECF179749126F6E7185E5EDA700EE9F9B79BC2D7989E6F28DD21CDB4832FA5"
+        intel_psw_url:       "https://oejenkins.blob.core.windows.net/oejenkins/Intel SGX PSW for Windows v2.9.100.2.exe"
+        intel_psw_hash:      "A2F357F3AC1629C2A714A05DCA14CF8C7F25868A0B3352FAE351B14AD121BDFC"
```

Comment:
I'm approving given the following change.

Could we had % signs in the URL here in place of spaces? It leads to random issues and is safer.